packages/cf-component-tabs/test/Tabs.js

```diff
@@ -39,18 +39,8 @@ describe('Tabs', function() {
         <div
           className="cf-tabs__panel"
           role="tabpanel"
-          aria-labelledby="1"
-          aria-hidden="true"
-          style={{ display: 'none' }}
-        >
-          One
-        </div>
-        <div
-          className="cf-tabs__panel"
-          role="tabpanel"
```

Comment:
@jwineman I didn't add specific tests since I feel like this is covered by the "will render" test. I can try to make a more explicit one if you'd prefer.